### PR TITLE
Adds terraform shortuts

### PR DESCRIPTION
**Pull request by ksatirli**
Hey @kibertoad,

recognizing that these shortcuts fall slightly outside your core area of interest, I'd still like a once-over from you.

The shortcuts are primarily for Terraform (with Docker, Git, and others added in future PRs) and install through `npm` (or `yarn`, if you're so inclined).

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (kibertoad) [inline .github/workflows/npm-publish.yml:11]: why ubuntu?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "terraform-untaint": "terraform-untaint.sh",
+    "terraform-workspace-delete": "terraform-workspace-delete.sh",
+    "terraform-workspace-new": "terraform-workspace-new.sh"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "Kerim Satirli <kerim@operatehappy.com>",
+  "license": "Apache-2.0"
+}
```
Comment (kibertoad) [inline package.json:30]: it might be helpful to include `files` field to define what needs to be published.

**Comment by kibertoad** [comment]
LGTM!

File: package.json (hunk 2 above)
Comment (ksatirli) [reply package.json:30]: Good one!

File: .github/workflows/npm-publish.yml (hunk 1 above)
Comment (ksatirli) [reply .github/workflows/npm-publish.yml:11]: This defines where the `nom publish` job as part of GitHub Actions runs on. 

Ubuntu was the default, so I kept it.

**Pull request merged**